### ISSUE TITLE
Tips: Adding a list of urls to download.

### ISSUE THREAD

**Issue by Qualzz** [StefanLobbenmeier/youtube-dl-gui#8]
Until this is officially implemented, here is a trick to add as many urls as once: 

First, close the app, go to the working folder of the app

On windows this is: `\AppData\Local\Programs\youtube-dl-gui`
Navigate to the `\resources\app.asar.unpacked` folder
Find the taskList file and open it with a text editor.

Add an array of all urls you want to download here. Exemple:
`["url1", "url2", "url3"]`

Restart the app, and voila 👯
The app will think that they were your previous download tasks to recover.

**Comment by StefanLobbenmeier** [comment]
That’s a clever trick to add the urls manually. Usually you should also be able to add a playlist, e.g. a YouTube playlist url or the link to a whole season on some sites and it will download all of them.

**Comment by Qualzz** [comment]
yes, another way is just to have all your url separated by a comma. 

url1, url2, url3

Paste this in the url textbox, and they should be parsed correctly
